src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -49,7 +52,12 @@ public sealed class QuicStream : System.IO.Stream
         public override void Write(byte[] buffer, int offset, int count) => throw null;
         public long StreamId => throw null;
         public void AbortRead() => throw null;
+        public void AbortWrite(long errorCode) => throw null;
         public ValueTask WriteAsync(ReadOnlyMemory<byte> data, bool endStream, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public ValueTask WriteAsync(ReadOnlySequence<byte> data, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public ValueTask WriteAsync(ReadOnlySequence<byte> data, bool endStream, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public ValueTask WriteAsync(ReadOnlyMemory<ReadOnlyMemory<byte>> data, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public ValueTask WriteAsync(ReadOnlyMemory<ReadOnlyMemory<byte>> data, bool endStream, System.Threading.CancellationToken cancellationToken = default) => throw null;
```

Comment:
Are these `endStream` parameters the only way to shutdown? How would I shutdown if have no more data to send?

Reply:
Only way right now (you can call WriteAsync with an empty buffer), I removed ShutdownAsync for now. I think we can bring it back.

Reply:
`HttpClient` will only ever know if it's the last write if there is no content being sent, so if `Shutdown()` would be more optimal than a full `WriteAsync` with empty buffer, that'd be good to have.

Based on how I would be able to use it, it'd give me better consistency to have a synchronous `Shutdown` -- this way you always call `Shutdown` or `WriteAsync(endStream: true)` followed by `WaitForShutdownCompleted`. This way user doesn't need to special-case based on which method initiated FIN.

Reply:
Alright, I'll add Shutdown for now.